src/components/ordinal-card/index.js

```diff
@@ -48,7 +48,12 @@ const OrdinalCard = ({ overlay, price, type, utxo, authors, confirmed, date, onS
 
     return (
         <div className={clsx("product-style-one", !overlay && "no-overlay")}>
-            <div className="card-thumbnail">{renderImage()}</div>
+            <div className="card-thumbnail">
+                {renderImage()}
+                <div className="inscription-details-area">
+                    <div className="inscription-number">No. {utxo.inscriptionNumber}</div>
```

Comment:
```suggestion
                    <div className="inscription-number">#{utxo.inscriptionNumber}</div>
```

Reply:
suggesting using `#` instead of `No.`

Reply:
also wondering if you can get the padding on the left to lineup with the rest of the items below it in the card